core/src/test/java/hudson/FunctionsTest.java

```diff
@@ -517,7 +520,8 @@ public void printStackTrace(PrintWriter s) {
         stack1.cause(stack2);
         stack2.cause(stack1);
         //Format changed in 11.0.9 / 8.0.272 (JDK-8226809 / JDK-8252444 / JDK-8252489)
-        if ((getVersion().getDigitAt(0) == 11 && getVersion().isNewerThanOrEqualTo(new VersionNumber("11.0.9"))) ||
```

Comment:
relax the check so that this also passes on newer versions.

12 and 13 still fail, 14 and 15 pass.

I don't think we need the tests to pass on 12 and 13 as they aren't supported.